src/ui/public/chrome/directives/global_nav/global_nav.js

```diff
@@ -46,8 +46,12 @@ module.directive('globalNav', (globalNavState, chrome) => {
         scope.isGlobalNavOpen = isOpen;
         scope.globalNavToggleButton = {
           classes: isOpen ? 'global-nav-link--close' : undefined,
-          title: isOpen ? 'Collapse' : 'Expand',
-          tooltipContent: isOpen ? 'Collapse side bar' : 'Expand side bar',
+          title: isOpen ?
+            i18n('common.ui.chrome.globalNav.collapseTitle', { defaultMessage: 'Collapse' })
```

Comment:
I would call it like .globalNav.navToggleButtonCollapseTitle. The same for other ids